opentracing-spring-web-starter/src/test/java/io/opentracing/contrib/spring/web/starter/SkipEndPointsWithContextPathWithoutBasePathTest.java

```diff
@@ -51,16 +51,15 @@
 @RunWith(SpringJUnit4ClassRunner.class)
 public class SkipEndPointsWithContextPathWithoutBasePathTest {
 
-  private static final String CONTEXT_PATH = "/myapp";
```

Comment:
Is there a test which sets a context path?

Reply:
@pavolloffay As pointed in the issue, since the TestRestTemplate always expands the context-path since `server.servlet.context-path:/myapp` all tests endpoints contain the context-path
![snap-4](https://user-images.githubusercontent.com/12292890/63106115-6e60c480-bf82-11e9-9159-d0dffe6d2426.png)